app/src/conferences/management/commands/create_data.py

```diff
@@ -235,53 +229,52 @@ class Command(BaseCommand):
 
     def handle(self, *args, **kwargs):
         if Zosia.objects.filter(active=True).count() > 0:
-            self.stdout.write('\033[1;91mThere is already active Zosia in database.'
+            self.stdout.write('\033[1;91mThere is already active ZOSIA in database.'
                               '\033[0m Do you want to create data anyway? [y/n]')
             choice = input().lower()
             if choice not in {'yes', 'y'}:
                 return
 
         place = create_place()
-        self.stdout.write('Place for zosia has been created!')
+        self.stdout.write('Place for ZOSIA has been created!')
 
         zosia = create_active_zosia(place)
-        self.stdout.write('Active zosia has been created!')
-
-        # for i in range(2):
-        #     create_past_zosia(place)
-        #     self.stdout.write('Past zosia #%d has been created' % i)
+        self.stdout.write('Active ZOSIA has been created!')
 
         all_users = []
 
         sample_staff_user = create_sample_staff_user()
-        self.stdout.write('Sample user has been created')
+        all_users.append(sample_staff_user)
+        self.stdout.write('Sample staff user has been created')
+
         sample_organizer_user = create_sample_organizer_user()
+        all_users.append(sample_organizer_user)
         self.stdout.write('Sample organizer has been created')
+
         create_contact_to_organizer(zosia, sample_organizer_user)
         self.stdout.write('Contact to sample organizer created')
-        all_users.append(sample_staff_user)
 
-        for i in range(5):
-            user_with_prefs = create_random_user_with_preferences(zosia, i + 1)
+        user_num = 5
+        for i in range(1, user_num + 1):
+            user_with_prefs = create_random_user_with_preferences(zosia, i)
             self.stdout.write(f"Created random user #{i}")
             all_users.append(user_with_prefs)
 
         for i in range(4):
             author = random.choice(all_users)
             create_lecture(zosia, author)
-            self.stdout.write(f"Created lecture #{i}")
+            self.stdout.write(f"Created lecture #{i + 1}")
 
         create_blogpost(sample_staff_user)
 
         question_num = random.randint(3, 9)
         for i in range(question_num):
             create_question()
-            self.stdout.write(f"Created question #{i}")
+            self.stdout.write(f"Created question #{i + 1}")
```

Comment:
In lines 258 and 275, you are starting loops from 1 to avoid incrementing the loop iterator inside the loop, and here you decided to do the opposite. I would suggest sticking to the previous convention, even though in those two i is referenced only once.

Reply:
I'll implement it in step 2.